src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -65,6 +83,24 @@ def schedule_auto_resolution():
 )
 @log_error_if_queue_has_items
 def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwargs):
+    use_workflow_engine = False
+
+    if use_workflow_engine:
```

Comment:
we might as well introduce the flag - especially since this is being used in a few different places.